### PR TITLE
Borrowed Books Page

### PR DESCRIPTION
**Pull request by ryankortbeek**
- added menu with a search icon
- implemented BorrowedBooksActivity
- implemented activity_borrowed.xml
- added support to BookList so that it is possible to configure whether book owner and/or book status is visible
- updated MyBooksActivity to be compatible with changes to BookList

Relevant issue(s):
- #30

### CODE REVIEW

File: src/app/src/main/java/com/example/bookmark/BorrowedActivity.java [hunk 1]
```diff
@@ -1,18 +1,99 @@
...
+ * borrowing. They can select a book which takes them to the
+ * BorrowedBookDetailsActivity where they can see the books details and
+ * return the book.
+ * TODO what else do we want here?
+ *
  * @author Ryan Kortbeek.
  */
 public class BorrowedActivity extends AppCompatActivity {
 
+    ArrayList<Book> borrowedBooks;
```
Comment (kylehennig) [inline src/app/src/main/java/com/example/bookmark/BorrowedActivity.java:34]: When declaring lists, we should prefer the generic List interface to the concrete ArrayList implementation. It makes it much easier to swap out the type of List we use internally in the future.
```suggestion
    List<Book> borrowedBooks;
```

File: src/app/src/main/java/com/example/bookmark/BorrowedActivity.java [hunk 2]
```diff
@@ -1,18 +1,99 @@
...
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_borrowed);
+
+        borrowedActionBar = getSupportActionBar();
+        assert borrowedActionBar != null;
+        borrowedActionBar.setTitle("Borrowed");
+
+        borrowedBooksListView = findViewById(R.id.borrowed_books_listview);
+
+        borrowedBooks = new ArrayList<>();
```
Comment (kylehennig) [inline src/app/src/main/java/com/example/bookmark/BorrowedActivity.java:51]: Could optionally move this to where borrowedBooks is first declared to reduce the visual distance.

File: src/app/src/main/res/layout/book_preview.xml [hunk 3]
```diff
@@ -23,7 +23,7 @@
         android:layout_marginStart="8dp"
         android:layout_marginTop="8dp"
         android:layout_marginEnd="16dp"
-        android:text="Book Title"
+        android:text="@string/book_title"
```
Comment (kylehennig) [inline src/app/src/main/res/layout/book_preview.xml:26]: Nice job making these shared string resources. A lot of pages will share our strings, so we want to continue doing this.

**Review by kylehennig — APPROVED** [review]

File: src/app/src/main/java/com/example/bookmark/BorrowedActivity.java (hunk 1 above)
Comment (ryankortbeek) [reply src/app/src/main/java/com/example/bookmark/BorrowedActivity.java]: Good thinking - I have made the change

**Review by kylehennig — APPROVED** [review]

**Pull request merged**